server/src/main/java/org/elasticsearch/node/Node.java

```diff
@@ -1055,6 +1055,8 @@ protected Node(
 
             logger.debug("initializing HTTP handlers ...");
             actionModule.initRestHandlers(() -> clusterService.state().nodesIfRecovered());
+            logger.debug("initializing operator handlers ...");
+            actionModule.initImmutableClusterStateHandlers(pluginsService);
```

Comment:
This doesn't seem like the right place in node construction to init these. The action module is here in the line above for other reasons (and can actually be moved up, it only needs the cluster service created). I would init these when the action module is created, as part of the ctor.